kclvm/tools/src/LSP/src/util.rs

```diff
@@ -30,3 +47,86 @@ pub fn get_file_name(vfs: RwLockReadGuard<Vfs>, file_id: FileId) -> anyhow::Resu
         ))
     }
 }
+
+pub fn get_compile_unit(file: &str) -> (Vec<String>, Option<LoadProgramOptions>) {
```

Comment:
Perhaps it would be more appropriate to put the compile unit into another package instead of the lsp util package. Add more comments and usage.